js_modules/dagster-ui/packages/ui-core/src/instigation/TickHistory.tsx

```diff
@@ -536,6 +538,15 @@ function TickRow({
           />
         </Box>
       </td>
+      <td>
+        <Button onClick={() => setShowLogs(true)}>View logs</Button>
+        <TickLogDialog
```

Comment:
I think this state may have been higher in the tree because new ticks can come in while you're viewing a tick and it goes off the current page / unmounts and closes? 

Would be nice to verify that you can open a tick and sit there for a while and the dialog doesn't close.

Reply:
Ah, interesting. I'll take a look.